parser/parser.y

```diff
@@ -715,6 +717,7 @@ import (
 	s3                    "S3"
 	schedule              "SCHEDULE"
 	staleness             "STALENESS"
+	startTime             "START_TIME"
```

Comment:
I noticed there is a `startTS` located below, do we need to use `startTS`?

Reply:
IMO, START_TIME is better

Reply:
I prefer start_time too. StartTs represent the pd txn timestamp in this context, but the start_time is a real datetime, better not to mix them.

Reply:
Makes sense